src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -770,9 +770,12 @@ private static String processPageSourceForGodMode(String content) {
         return content
                 .replaceAll("<#comment[ ]*</#comment>", "<!---->")
                 .replace(Config.APP_URL, "${app.url}")
-                .replace("http://localhost:8888", "${web.url}")
-                .replace(Config.APP_URL.replace("http", "https"), "${web.url}")
-                .replace("\"/_ah", "\"${web.url}/_ah")
+                .replace(TestProperties.inst().TEAMMATES_URL, "${test.url}")
+                .replace(TestProperties.inst().TEAMMATES_URL.replace("http", "https"), "${test.url}")
```

Comment:
yup, in one case they use javascript to get the page url which returns the https version rather than http version